### PR TITLE
[com_redirect] Solve javascript error on bulk import modal close

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for New Issue.
#### Summary of Changes

There is a javascript error on closing the "Bulk Import" modal of com_redirect.
This simple PR fixes it.
#### Testing Instructions

Code review, or:
1. Use latest staging
2. Open dev console on chrome
3. Go to Components -> Redirects and press "Bulk Import", the modal opens
4. Press "Cancel" on the modal, you'll see a javascript error on chrome console
5. Apply patch.
6. Repeat test, no js error.

### CODE REVIEW

File: administrator/components/com_redirect/views/links/tmpl/default_batch.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		</div>
 	</div>
 	<div class="modal-footer">
-		<button class="btn" type="button" onclick="document.id('batch_urls').value=''" data-dismiss="modal">
+		<button class="btn" type="button" onclick="document.getElementById('batch_urls').value='';" data-dismiss="modal">
```
Comment (JoomliC) [inline administrator/components/com_redirect/views/links/tmpl/default_batch.php]: Well... just a code style for attributes order : <code>data-*</code> after <code>class</code>
JCS: https://joomla.github.io/coding-standards/?coding-standards/chapters/html.md
And a more detailled list of attributes, which could be a good example (as by one of the creator of Bootstrap) : http://codeguide.co/#html-reducing-markup ;-)

**Comment by andrepereiradasilva** [comment]
it was already like that, but ok i changed.

**Comment by JoomliC** [comment]
> it was already like that, but ok i changed.

Yes, i know ;-)
But maybe good to start using code standards for all modified lines :smile:

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 828baee3dc60d604501171a25cf8a2d878d6c2aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10687">issues.joomla.org/joomla-cms/10687</a>.</sub>

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on 828baee3dc60d604501171a25cf8a2d878d6c2aa<br /><br />Firebug for FF detects this error too.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10687">issues.joomla.org/joomla-cms/10687</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10687">issues.joomla.org/joomla-cms/10687</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**